src/features/game/types/game.ts

```diff
@@ -540,6 +540,7 @@ export type Mine = Position;
 export type BuildingProduct = {
   name: CookableName;
   readyAt: number;
+  amount: Decimal;
```

Comment:
I would consider keeping this as a `number`.

There is some strange behaviour when persisting `Decimal` types - unless absolutely needed, we opt for storing just as number.

Also, I would suggest making this an optional value so it is backwards compatible with all of the current recipes that are already cooking.

`amount?: number`